custom_components/google_home/const.py

```diff
@@ -54,13 +55,14 @@
 # API
 API_ENDPOINT_ALARMS = "setup/assistant/alarms"
 API_ENDPOINT_DELETE = "setup/assistant/alarms/delete"
+API_ENDPOINT_REBOOT = "setup/reboot"
 HEADER_CAST_LOCAL_AUTH = "cast-local-authorization-token"
 HEADER_CONTENT_TYPE = "content-type"
 
 # HEADERS
 HEADERS = {
     HEADER_CAST_LOCAL_AUTH: "",
-    HEADER_CONTENT_TYPE: "application/json; charset=UTF-8",
+    HEADER_CONTENT_TYPE: "application/json",
```

Comment:
Probably this is default but I remember you said you can assign name to an alarm? Can you check if UTF-8 name works fine without setting `charset`?

Reply:
It works, i even tried with the special letters we have here in Denmark and it works just fine. It was maybe me who added it sometime ago with out fully understanding that it was not needed. 😄 

[`Ghlocalapi`](https://rithvikvibhu.github.io/GHLocalApi/#section/Google-Home-Local-API/Getting-Started) also does not say anything about it 😄